### ISSUE TITLE
No config file

### ISSUE THREAD

**Issue by kubabogaczewicz** [onoclea/admin-ldap-password#1]
Even though it's small why not extract ldap settings into config.rb? :)

**Comment by sawicki-eu** [comment]
True, true :) Parsing YAML files is not a rocket science so I suppose this will come sooner or later.

Thanks for your imput.

Cheers,
